lib/Jsrt/ChakraCommon.h

```diff
@@ -1040,9 +1040,9 @@ typedef unsigned char* ChakraBytePtr;
     /// </returns>
     CHAKRA_API
         JsParseScript(
-            _In_z_ const wchar_t *script,
+            _In_z_ const CSTR_TYPE *script,
```

Comment:
Since they are win32 only, they should be #ifdef out for non-Win32, not typedef CSTR_TYPE and appearing cross-platform.